### DISCUSSION TITLE
付款后没有开通

### DISCUSSION THREAD

**Discussion by louzhiqing1990** [getlantern/lantern-forum-cn#585] [Q&A]
购买了一年会员，邮箱号louzhiqing@126.com，没有收到激活码

**Comment by louzhiqing1990** [comment]
@Mike777Khoo

**Comment by MaxMKMax** [comment]
@louzhiqing1990 您好. 请您也提供我支付单据. 已经帮您提交问题了. 但负责人要周二才回来. 我会帮您追问的. 不好意思让您久等了. 请您过 2-3 天再查看这个帖子或邮箱. 我们会给您发送激活码. 谢谢您支持蓝灯!

**Reply by louzhiqing1990** [reply]
![Uploading image.png…]()
![Uploading IMG_1216.PNG…]()

**Comment by MaxMKMax** [comment]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/2c54efd4-cfea-431d-a423-2935bd35619c)

@louzhiqing1990 您好. 请您发邮件给我吧. 我这里看不到您的截图. 被Github屏蔽了.
mike@getlantern.org

**Comment by louzhiqing1990** [comment]
已发邮箱，请尽快处理

**Reply by MaxMKMax** [reply]
@louzhiqing1990 
https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.

**Reply by louzhiqing1990** [reply]
按链接步骤操作了，什么时候能解决？
